src/gameLogic/generation/ActionGenerator.hpp

```diff
@@ -101,10 +102,11 @@ namespace spy::gameplay {
             generateLaserCompact(const State &s, const util::UUID &activeCharacter, const spy::MatchConfig &config);
 
             static std::vector<std::shared_ptr<BaseOperation>>
-            generateMirrorOfWilderness(const State &s, const util::UUID &activeCharacter, const spy::MatchConfig &config);
+            generateMirrorOfWilderness(const State &s, const util::UUID &activeCharacter,
+                                       const spy::MatchConfig &config);
```

Comment:
Der MirrorOfWilderness hat in der MatchConfig lediglich die Erfolgswahrscheinlichkeit, für die Generierung der Aktion sollte die unerheblich sein